src/slow_log/slow_log.c

```diff
@@ -13,14 +13,24 @@
 #include "../util/rmalloc.h"
 #include "../util/thpool/thpool.h"
 
+extern threadpool _readers_thpool;  // declared in module.c
+extern threadpool _writers_thpool;  // declared in module.c
+
 static int get_thread_id() {
```

Comment:
Add explanatory comment like:
```
/* get_thread_id returns 0 for the Redis main thread,
  * reader thread count + 1 for the writer thread,
  * or reader thread ID for a reader thread. */
```

We should consider relocating this function to `thpool.c`, since it gets invoked here and by `command_ctx.c` (though thankfully nowhere else).